processors/xxhash.go

```diff
@@ -0,0 +1,43 @@
+package processors
+
+import (
+	"fmt"
+
+	"github.com/cespare/xxhash/v2"
+)
+
+type XXHash64 struct{}
+
+func (x XXHash64) Name() string {
+	return "xxhash-64"
```

Comment:
It should be XXH64 here, but you could add aliases maybe.

But maybe it's a matter of consistency with existing command